### PR TITLE
Soc/46 display offers

### PR DESCRIPTION
**Pull request by yimialmonte**
# What changed:
- Show offers on home page
- Loading 

# How to Test / Acceptance Criteria:
- Go home it should show offers 

# Ticket(s): 
- 46

# checklist
- [] Tests
- [] Documentation

# Additional Info
- This offers come from backend fake data sample

### CODE REVIEW

File: src/Pages/Home.js [hunk 1]
```diff
@@ -1,18 +1,37 @@
...
 
 class Home extends Component {
   constructor(props) {
     super(props);
     this.state = {};
   }
 
+  componentDidMount() {
+    const { fetchoffers } = this.props;
+    fetchoffers();
```
Comment (cesar18pena) [inline src/Pages/Home.js:17]:
```suggestion
    this.props.fetchoffers();
```

File: src/components/Offers.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      {offers && offers.map((offer) => (
+        <Col sm={12} md={6} lg={4} key={offer._id}>
+          <OfferDetails offer={offer} />
+        </Col>
+      ))}
+    </Row>
+  );
+}
+
+Offers.propTypes = {};
```
Comment (cesar18pena) [inline src/components/Offers.js:18]: You are missing the PropTypes validation here

File: src/components/OfferDetails.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          alt="Card image cap"
+        />
+      </Link>
+      <CardBody>
+        <CardTitle tag="h5">{offer.name}</CardTitle>
+        <CardSubtitle tag="h6" className="mb-2 text-muted">
+          {offer.brand}
+        </CardSubtitle>
+        <CardText>
+          <big className="text-danger">Discount: -{offer.percentDiscount}%</big>
```
Comment (cesar18pena) [inline src/components/OfferDetails.js]: This is an unsupported tag of HTML; so use another tag or a React component https://www.w3schools.com/tags/tag_big.asp

File: src/components/Offers.js (hunk 2 above)
Comment (yimialmonte) [reply src/components/Offers.js:18]: Fixed

File: src/components/OfferDetails.js (hunk 3 above)
Comment (yimialmonte) [reply src/components/OfferDetails.js]: Fixed

File: src/Pages/Home.js (hunk 1 above)
Comment (yimialmonte) [reply src/Pages/Home.js:17]: Make sense of the suggestion but eslint got me this warning: 'Must use destructuring props assignment'

**Review by cesar18pena — APPROVED** [review]

**Pull request merged**